kernel/kernel-keymanager-service/pom.xml

```diff
@@ -17,7 +17,7 @@
 		<!-- maven -->
 		<maven.compiler.source>21</maven.compiler.source>
 		<maven.compiler.target>21</maven.compiler.target>
-		<maven.compiler.version>3.9.6</maven.compiler.version>
+		<maven.compiler.version>3.9.0</maven.compiler.version>
```

Comment:
This change had to be done as 3.9.6 plugin was not available.

ref: https://repo1.maven.org/maven2/org/apache/maven/plugins/maven-compiler-plugin/3.9.6/